ui/components/Terminal.jsx

```diff
@@ -28,19 +28,19 @@ const TerminalBackdrop = styled.div`
   top: 0;
   left: 0;
   opacity: ${props => (props.show ? 1 : 0)};
-  pointer-events: ${props => (props.show ? 'all' : 'none')};
+  visibility: ${props => (props.show ? 'visible' : 'hidden')};
   transition: 300ms;
 `;
-const Terminal = ({ text, show, hide }) => (
+const Terminal = ({ text, show, onBackdropClick }) => (
   <React.Fragment>
-    <TerminalBackdrop onClick={hide} show={show} />
+    <TerminalBackdrop onClick={onBackdropClick} show={show} />
```

Comment:
you can use `hidden` attribute instead of toggling visibility for better a11y, or not rendering it at all 🤔

Reply:
Without rendering it you can't get the cool fade-in, out transition :sweat_smile:.

Reply:
Just joking, i will change this

Reply:
didn't think of the transitions, if we later want to add them we can switch it to something in CSS I guess?

Reply:
you can't transition the ```hidden``` attribute or the rendering. That's why i added ```visibility: none``` in the CSS

Reply:
i don't find any reason why ```visibility: hidden``` is bad, it should work correctly on screen readers